src/main/java/org/commcare/core/network/ModernHttpRequester.java

```diff
@@ -143,6 +155,7 @@ private Response executeAndCheckCaptivePortals(Call currentCall) throws IOExcept
 
     public static void processResponse(HttpResponseProcessor responseProcessor,
                                        int responseCode,
+                                       String errorMessage,
```

Comment:
this is unused at this point of time, would be nice to see how you are actually using it. Though extracting it and sending it to the response processor in case of errors sounds right to me as a change.

Reply:
Yeah, I wasn't sure how far to pass the message up the chain so stopped here. In practice I'm only using it during debugging, as some errors from the ConnectID server are coming back as strings. So when I need to debug a failed server call I just set a breakpoint in the error case and read the errorMessage that way.

But in the long run I don't think the plan is for the server to return error messages as text. Instead we'll have the server return error codes that can be converted to translatable strings (for localization). So I'm open to suggestions as far as how to proceed with this change. Maybe it's not worth keeping around in the long run? Or we could log the errorMessage somehow when it's not empty as a way to relevantly use the code?

Reply:
Yeah I don't think we want to add any code just for debugging, you should be able to navigate in debugger to the caller of this method to look at `response.errorBody()` directly through value inspection in debugger.

Reply:
No problem, I'll revert this code.